app/src/main/res/values/dimens.xml

```diff
@@ -26,4 +26,7 @@
     <dimen name="tileLetterFontSize">66sp</dimen>
     <dimen name="tileLetterFontSizeForShortcut">48dp</dimen>
     <dimen name="tileLetterImageSize">92dp</dimen>
+
+    <!--    Margin for EditText inside MaterialAlertDialog-->
+    <dimen name="dialog_margin">20dp</dimen>
```

Comment:
I think this can be removed now that it's no longer used.

Reply:
Yes, I forgot to remove it. I'll remove it